ci/mobile.groovy

```diff
@@ -0,0 +1,76 @@
+common = load 'ci/common.groovy'
+
+def uploadArtifact() {
+  def artifact_dir = pwd() + '/android/app/build/outputs/apk/release/'
+  println (artifact_dir + 'app-release.apk')
+  def artifact = (artifact_dir + 'app-release.apk')
+  def server = Artifactory.server('artifacts')
+  shortCommit = sh(
+    returnStdout: true,
+    script: 'git rev-parse HEAD'
+  ).trim().take(6)
+  def filename = "im.status.ethereum-${shortCommit}-n-fl.apk"
+  def newArtifact = (artifact_dir + filename)
+  sh "cp ${artifact} ${newArtifact}"
+  def uploadSpec = '{ "files": [ { "pattern": "*apk/release/' + filename + '", "target": "nightlies-local" }]}'
+  def buildInfo = server.upload(uploadSpec)
+  return 'http://artifacts.status.im:8081/artifactory/nightlies-local/' + filename
+}
+
+def prepDeps() {
+  sh 'rm -rf node_modules'
+  sh 'cp .env.nightly .env'
+  common.installJSDeps('mobile')
+  sh 'mvn -f modules/react-native-status/ios/RCTStatus dependency:unpack'
+  /* generate ios/StatusIm.xcworkspace */
+  dir('ios') {
+    sh 'pod install'
+  }
+}
+
+def bundleAndroid() {
+  withCredentials([
+    string(credentialsId: "SUPPLY_JSON_KEY_DATA", variable: 'GOOGLE_PLAY_JSON_KEY'),
+    string(credentialsId: "SLACK_URL", variable: 'SLACK_URL')
+  ]) {
+    sh 'bundle exec fastlane android nightly'
```

Comment:
this one uploads to the play store...

any place where we upload to diawi?

Reply:
yeah, in the `uploadiOS()` method